### PR TITLE
[VEN-2623] Add setInterestRateModel to the VBNBAdmin contract

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: contracts/Admin/VBNBAdminStorage.sol [hunk 1]
```diff
@@ -9,6 +9,8 @@ interface VTokenInterface {
     function _acceptAdmin() external returns (uint);
 
     function comptroller() external returns (address);
+
+    function _setInterestRateModel(address newInterestRateModel) external returns (uint);
```
Comment (chechu) [inline contracts/Admin/VBNBAdminStorage.sol:13]: This function is `public` in VToken.sol

File: contracts/Admin/VBNBAdmin.sol [hunk 2]
```diff
@@ -91,6 +91,17 @@ contract VBNBAdmin is ReentrancyGuardUpgradeable, AccessControlledV8, VBNBAdminS
...
 
+    /**
+     * @notice Sets the interest rate model of the vBNB contract
+     * @param newInterestRateModel Address of the new interest rate model
+     * @custom:access Controlled by ACM
+     */
+    function setInterestRateModel(address newInterestRateModel) external {
+        _checkAccessAllowed("_setInterestRateModel(address)");
+        uint256 err = vBNB._setInterestRateModel(newInterestRateModel);
+        require(err == 0, "setting interest rate model failed");
```
Comment (chechu) [inline contracts/Admin/VBNBAdmin.sol:102]: I would not require anything here, I would just return the value returned by `vBNB._setInterestRateModel`, as we are doing in the fallback function

File: deployments/bscmainnet.json
Comment (chechu) [inline deployments/bscmainnet.json:1]: We also need the deployment to bsctestnet, and the associated VIP [here](https://github.com/VenusProtocol/vips/pull/297)

**Review by chechu — APPROVED** [review]

**Review by kkirka — APPROVED** [review]
Approved, though the PR title is misleading, can we change that?

**Pull request merged**